common-test-utils/pom.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus.qe</groupId>
+        <artifactId>beefy-scenarios</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>common-test-utils</artifactId>
+
+    <dependencies>        
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>kafka</artifactId>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-test-common</artifactId>
```

Comment:
This makes this repo tightly coupled with infrastructure module in main Quarkus repo.
I do not like it too much. What are the benefits of this approach?

Reply:
This is to have the QuarkusTestResource annotation into this module. We can use the quarkus junit5 dependency instead.

Reply:
I prefer to use things intended for end users, quarkus-test-common is not mentioned in quarkus docs / guides at all.

It's a helper in main Quarkus repo and I would keep it that way. Apps in this repo should try to avoid such helpers as they can pull in a lot of stuff and can hide something that way.

Reply:
Will remove this module